bootstrapping-lambda/src/util.ts

```diff
@@ -1,4 +1,42 @@
+import * as AWS from "aws-sdk";
 import { Response } from "express";
+import { STAGE, standardAwsConfig } from "../../shared/awsIntegration";
+import { pbkdf2Sync } from "crypto";
+
+const ssm = new AWS.SSM(standardAwsConfig);
+
+// memoise fetched salt - memo is invalidated at midnight
+// a failed fetch _is also memoised_!! this avoids lots of fetches if one fetch fails
+const saltmemo = (async () => {
+  let lastFetch = new Date();
+  let salt: Buffer | undefined;
+
+  const get = async () => {
+    if (lastFetch.getUTCDate() === new Date().getUTCDate()) {
+      return salt;
+    }
+    const saltParameter = ssm
```

Comment:
I think this could be done with the existing param store retrieval function https://github.com/guardian/pinboard/blob/bab6d9e3a5208541a6b96200bddacb1e3f9c93ed/shared/awsIntegration.ts#L40